bgpd/bgp_route.c

```diff
@@ -529,6 +529,9 @@ static uint32_t bgp_med_value(struct attr *attr, struct bgp *bgp)
 void bgp_path_info_path_with_addpath_rx_str(struct bgp_path_info *pi, char *buf,
 					    size_t buf_len)
 {
+	if (!pi || !pi->peer)
```

Comment:
how is a pi created without a peer?  Why are we testing for it?

Reply:
Fixed.